tardis/plasma/standard_plasmas.py

```diff
@@ -73,7 +73,15 @@ def assemble_plasma(config, model, atom_data=None):
             raise ValueError('No atom_data option found in the configuration.')
 
         logger.info('Reading Atomic Data from %s', atom_data_fname)
-        atom_data = atomic.AtomData.from_hdf(atom_data_fname)
+
+        try:
+            atom_data = atomic.AtomData.from_hdf(atom_data_fname)
+        except TypeError as e:
```

Comment:
and no `\` 😉

Reply:
okay, I'll have another go.. so catch the error, access the message and display it with the helpful suggestion or possible reason

Reply:
oh..now I see